### PR TITLE
Wizard rererebased

### PR DESCRIPTION
**Pull request by thisconnect**
in favor of https://github.com/digitalbitbox/bitbox-wallet-app/pull/28

### CODE REVIEW

**Comment by ferdzbautistaolazo** [comment]
Ok.thanks a lot for the update.

On Sep 6, 2018 16:11, "tcme" <notifications@github.com> wrote:

in favor of #28 <https://github.com/digitalbitbox/bitbox-wallet-app/pull/28>
------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/digitalbitbox/bitbox-wallet-app/pull/54
Commit Summary

   - frontend: wizard updates
   - frontend: update wizard ui
   - wizard ui spacing tweaks
   - frontend: wizard updates
   - frontend: success screen
   - frontend: uniform back buttons / alert updates / spacing adjustments

File Changes

   - *A* frontends/web/src/assets/icons/ok.svg
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-0>
   (2)
   - *A* frontends/web/src/assets/icons/warning.png
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-1>
   (0)
   - *M* frontends/web/src/components/backups/backups.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-2>
   (6)
   - *M* frontends/web/src/components/backups/backups.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-3>
   (38)
   - *M* frontends/web/src/components/backups/restore.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-4>
   (5)
   - *M* frontends/web/src/components/footer/footer.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-5>
   (4)
   - *M* frontends/web/src/components/footer/footer.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-6>
   (3)
   - *A* frontends/web/src/components/icon/assets/icons/alert-triangle.svg
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-7>
   (1)
   - *A* frontends/web/src/components/icon/icon.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-8>
   (24)
   - *A* frontends/web/src/components/icon/index.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-9>
   (2)
   - *M* frontends/web/src/components/message/message.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-10>
   (25)
   - *M* frontends/web/src/components/message/message.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-11>
   (7)
   - *M* frontends/web/src/components/spinner/Spinner.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-12>
   (8)
   - *M* frontends/web/src/components/wait-dialog/wait-dialog.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-13>
   (2)
   - *M* frontends/web/src/i18n/en.json
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-14>
   (84)
   - *M* frontends/web/src/routes/device/device.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-15>
   (61)
   - *M* frontends/web/src/routes/device/device.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-16>
   (33)
   - *A* frontends/web/src/routes/device/manage-backups/manage-backups.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-17>
   (7)
   - *M* frontends/web/src/routes/device/manage-backups/manage-backups.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-18>
   (23)
   - *M* frontends/web/src/routes/device/setup/components/steps.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-19>
   (75)
   - *M* frontends/web/src/routes/device/setup/components/steps.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-20>
   (75)
   - *M* frontends/web/src/routes/device/setup/goal.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-21>
   (50)
   - *M* frontends/web/src/routes/device/setup/initialize.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-22>
   (63)
   - *A* frontends/web/src/routes/device/setup/security-information.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-23>
   (101)
   - *M* frontends/web/src/routes/device/setup/seed-create-new.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-24>
   (74)
   - *M* frontends/web/src/routes/device/setup/seed-restore.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-25>
   (87)
   - *A* frontends/web/src/routes/device/setup/success.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-26>
   (43)
   - *M* frontends/web/src/routes/device/unlock.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-27>
   (78)
   - *M* frontends/web/src/routes/device/waiting.jsx
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-28>
   (24)
   - *M* frontends/web/src/style/layout.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-29>
   (12)
   - *M* frontends/web/src/style/variables.css
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-30>
   (2)
   - *M* frontends/web/src/utils/innerHTML.js
   <https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-31>
   (4)

Patch Links:

   - https://github.com/digitalbitbox/bitbox-wallet-app/pull/54.patch
   - https://github.com/digitalbitbox/bitbox-wallet-app/pull/54.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/digitalbitbox/bitbox-wallet-app/pull/54>, or mute the
thread

**Comment by thisconnect** [comment]
hey @x1ddos please review this :)

File: frontends/web/src/routes/device/setup/initialize.jsx [hunk 1]
```diff
@@ -95,20 +95,26 @@ export default class Initialize extends Component {
...
-                    onClick={goBack}>
-                    {t('button.back')}
-                </Button>
+            <div className={style.block}>
+                <h2>{t('initialize.info.subtitle')}</h2>
+                <ul>
+                    <InnerHTMLHelper tagName="li" html={t('initialize.info.description_1')} />
+                    <InnerHTMLHelper tagName="li" html={t('initialize.info.description_2')} />
+                </ul>
+                <InnerHTMLHelper tagName="p" html={t('initialize.info.description_3')} />
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/routes/device/setup/initialize.jsx]: Yes this is so we can have simple inline html markup in the translation files, for example here https://github.com/digitalbitbox/bitbox-wallet-app/pull/54/files#diff-542f9d5d7fa6c4086dc202220e52e223R370

File: frontends/web/src/routes/device/setup/seed-create-new.jsx [hunk 2]
```diff
@@ -172,7 +180,10 @@ export default class SeedCreateNew extends Component {
                         onValidPassword={this.setValidBackupPassword} />
                 </div>
                 <div class={style.agreements}>
-                    <p>{t('seed.description')}</p>
+                    <div class="flex flex-row flex-start flex-items-center">
+                        <img src={warning} style="width: 18px; margin-right: 10px; position: relative; bottom: 1px;" />
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/routes/device/setup/seed-create-new.jsx]: Agree, but it is ok for you, we should factor this into it's own component and reuse it where we need warning/messages. However this PR is already too big. I'd love to do this in an different PR

**Comment by thisconnect** [comment]
squashed and force pushed

**Comment by x1ddos** [comment]
but why do you create a new PR each time? :)
I think it'd be easier to keep the same pull request, which would also preserve comments.

**Comment by thisconnect** [comment]
> but why do you create a new PR each time? :)
> I think it'd be easier to keep the same pull request, which would also preserve comments.

yes agree

**Pull request closed without merging**